data/questions/fr/disquetusais.js

```diff
@@ -261,7 +261,7 @@ export default [
     },
     correctAnswers: ['c'],
     explanation:
-      "Ces durées sont mentionnées dans l'annexe des règles A5.4.1 (https://rules.wfdf.org/documents/wfdf-rules-of-ultimate-2021-2024-appendix-v2-0/download)",
+      "Ces durées sont mentionnées dans l'annexe des règles A5.4.1 (https://rules.wfdf.org/documents/wfdf-rules-of-Ultimate-2021-2024-appendix-v2-0/download)",
```

Comment:
A vrai dire j'ai l'impression qu'une v3 est parue, donc l'url est maintenant https://rules.wfdf.org/documents/wfdf-rules-of-ultimate-2021-2024-appendix-v3/download

Reply:
Fixed